### ISSUE TITLE
[BUG] watson crashes at start after some update

### ISSUE THREAD

**Issue by Niedzwiedzw** [TailorDev/Watson#451]
```bash
❯ watson status
Traceback (most recent call last):
  File "/usr/bin/watson", line 5, in <module>
    from watson.__main__ import cli
  File "/usr/lib/python3.9/site-packages/watson/__main__.py", line 1, in <module>
    from watson import cli
  File "/usr/lib/python3.9/site-packages/watson/cli.py", line 529, in <module>
    def report(watson, current, from_, to, projects, tags, ignore_projects,
  File "/home/niedzwiedz/.local/lib/python3.9/site-packages/click/decorators.py", line 247, in decorator
    _param_memo(f, OptionClass(param_decls, **option_attrs))
  File "/usr/lib/python3.9/site-packages/watson/cli.py", line 46, in __init__
    super(MutuallyExclusiveOption, self).__init__(*args, **kwargs)
  File "/home/niedzwiedz/.local/lib/python3.9/site-packages/click/core.py", line 2482, in __init__
    super().__init__(param_decls, type=type, multiple=multiple, **attrs)
  File "/home/niedzwiedz/.local/lib/python3.9/site-packages/click/core.py", line 2108, in __init__
    raise ValueError(
ValueError: 'default' must be a list when 'multiple' is true.
```

**Comment by ericguirbal** [comment]
Until this bug is fixed, you must downgrade click to version 7.1.2.:  `pip install --click==7.1.2`.

**Comment by davinellulinvega** [comment]
This is related to both issue #443 and pull request #432. If you want more info on why this is happening and how you can work around it in the meantime, have a look at those.

**Comment by michaelaye** [comment]
I don't understand how click is a not fitting version when I installed td-watson via pipx. Does that mean it itself doesn't list click=7.1.2 as a requirement?

**Comment by aparkerlue** [comment]
For pipx installations, you can inject the appropriate version of click into the pipx-managed virtual environment:
```sh
pipx inject td-watson click==7.1.2
```

**Comment by jmaupetit** [comment]
Should be fixed with the 2.1.0 fresh release!
